example/lib/leaflet-fullUrlHash.js

```diff
@@ -0,0 +1,198 @@
+(function(window) {
+	var HAS_HASHCHANGE = (function() {
+		var doc_mode = window.documentMode;
+		return ('onhashchange' in window) &&
+			(doc_mode === undefined || doc_mode > 7);
```

Comment:
I think we need to add this to a separate folder outside example. Let's make one lib folder at the root level and  change the script tag above.

Reply:
Should I move the whole lib folder to the root or just the fullUrlHash.js file?